### DISCUSSION TITLE
Vercel: This Serverless Function (FUNCTION_INVOCATION_FAILED) with pocket host

### DISCUSSION THREAD

**Discussion by hybornconcept** [pockethost/pockethost#286] [Q&A]
I have a sveltekit project, that is perfecly rnning on my local machine. I use PocketBase as my DB, which is hosted on pockethost. The Pockethost domain is saved in a .env file locally  and I have  set the vercel env settings properly. As soon as vercel has built the project, I see the following Error: FUNCTION_INVOCATION_FAILED. The logs say, that they cannot GET /

Of course I have googled and tried to resolve the issue for example by temporary deleting the images from  the static folder. Bt that didn't help. 

I have no Idea, what the Problem is. Does somebody have any ideas?

**Comment by benallfree** [comment]
Can you paste a screenshot of the exact network log showing the error?
